cider/function/aggregate/GroupByRuntime.cpp

```diff
@@ -67,7 +67,7 @@ extern "C" RUNTIME_EXPORT NEVER_INLINE int64_t* get_group_value_cider(
   return agg_hash_table->getGroupTargetPtr(key);
 }
 
-extern "C" RUNTIME_EXPORT ALWAYS_INLINE int32_t
+extern "C" RUNTIME_EXPORT ALWAYS_INLINE int64_t
```

Comment:
Why do this change?

Reply:
It should be `int64_t` since we get that value from `lookupIdByValue`.